app/decorators/service_decorator.rb

```diff
@@ -56,4 +56,30 @@ def as_json(options = {})
   def backend_api?
     false
   end
+
+  def links
+    [
+      { name: 'Edit', path: '' },
+      { name: 'Overview', path: h.admin_service_path(object) },
+      { name: 'Analytics', path: h.admin_service_stats_usage_path(object) },
+      { name: 'Applications', path: h.admin_service_applications_path(object) },
+      { name: 'ActiveDocs', path: h.admin_service_api_docs_path(object) },
+      { name: 'Integration', path: h.path_to_service(object) },
+    ]
+  end
+
+  def apps_count
+    cinstances.size
+  end
+
+  def backends_count
+    backend_apis.size
```

Comment:
inverse situation to https://github.com/3scale/porta/pull/2194/files#r501110645

Reply:
Would `service.backend_api_configs` be valid here, like it's in `Account > Backends`?
https://github.com/3scale/porta/blob/ed7ad0d57ab2c8de7d4241afd5008f89de31fc59/app/controllers/api/backend_usages_controller.rb#L16